components/calendar/index.css

```diff
@@ -40,23 +42,26 @@ governing permissions and limitations under the License.
 
   line-height: var(--spectrum-calendar-title-height);
   margin: 0;
-  padding: 0 var(--spectrum-calendar-title-height);
   order: 1;
   flex-grow: 1;
 
+
   text-align: center;
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
 }
 
+.spectrum-Calendar-prevMonth,
+.spectrum-Calendar-nextMonth {
+  margin: 0 var(--spectrum-calendar-button-gap) !important; /* override adjacent sibling selector from ActionButton */
```

Comment:
There was a task for Spectrum CSS to remove the !important bangs. Is there maybe a different solution possible to se the correct margin for prev-Month, next-Month?

Reply:
Yes, the only solution is to increase selector specificity. I went ahead and did that.